P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1330 @@
+            type="hashtag" target="https://twitter.com/hashtag/dynamicduo?src=hash"
+            >#dynamicduo</ref>
+          <ref type="hashtag"
+            target="https://twitter.com/hashtag/wirk%C3%BCmmernunsauchumIhrenEmpfang?src=hash"
+            >#wirkümmernunsauchumIhrenEmpfang</ref>
+          <ref type="hashtag" target="https://twitter.com/hashtag/bcrn18?src=hash">#bcrn18</ref>
+          <ref type="hashtag" target="https://twitter.com/hashtag/wikidach?src=hash"
+            >#wikidach</ref>
+        </post>
+        <post modality="written" generatedBy="human" type="tweet" who="#u2"
+          synch="#tweetsbcrn18.t003" xml:lang="de" xml:id="cmc_post_1043796093786566656">
+          <time generatedBy="system"> 14:35 </time> Immer wieder gerne. Kann ich mich schon für
+          nächstes Jahr als Empfangs- <ref type="hashtag"
+            target="https://twitter.com/hashtag/Engel?src=hash">#Engel</ref> für das nächste
+          BarCamp bewerben <w pos="EMO">&#x1F92A;</w>
+          <ref type="hashtag" target="https://twitter.com/hashtag/bcrn18?src=hash">#bcrn18</ref>
+          <trailer>
+            <fs>
+              <f name="favoritecount">
+                <numeric value="4"/>
+              </f>
+            </fs>
+          </trailer>
+        </post>
+      </egXML>
+      <p>Note that in the above example <soCalled>CoMeRe</soCalled> style (cf. <ptr
+      target="#BIB_CoMeRe"/>) encoding is used to represent the number of favorites. It would
+      also be reasonable to use a TEI <gi>measure</gi> element instead of the <gi>fs</gi>.</p>
+    </div>
+    <div xml:id="CMCanalysis">
+      <head>Linguistic Annotation</head>
+      <p>For encoding linguistic analyses of CMC text, we may use the dedicated elements and
+        attributes from the analysis module, which is described in <ptr target="#SA"/>. For example,
+        the tokenisation (segmentation into word-like units) of a CMC text should be encoded using
+        the <gi>w</gi> element.</p>
+      <p>Let us take, for example a posting that contains the content <q xml:lang="de">00:22 Bin
+          soooooo im stress gewesen ich Armer lol</q> (in English: <q>I was soooooo stressed out
+          poor me lol</q>). This may be encoded as follows. <!-- aus Dortmunder Chatkorpus Chat 1101004_Welcome_2004-11-08.tei.xml -->
+        <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de">
+          <post generatedBy="human" rend="color:black" synch="#t010.a" who="#A03.a"
+            xml:id="cmc_post_m16.a">
+            <time>00:22</time>
+            <w>Bin</w>
+            <w>soooooo</w>
+            <w>im</w>
+            <w>stress</w>
+            <w>gewesen</w>
+            <w>ich</w>
+            <w>Armer</w>
+            <w>lol</w>
+          </post>
+        </egXML>
+      </p>
+      <p>In many CMC genres, especially in private chat, sloppy and creative writing abound
```

Comment:
```suggestion
      <p>In many CMC genres, especially in private chat, casual and creative writing abound
```

Reply:
I would replace “sloppy” with “casual” because “sloppy” seems pejorative, but I let English native speakers decide whether the suggestion should be committed.

Reply:
@HelenaSabel I agree, and the word "sloppy" also seems (literally) sloppy to me. I simplified it a little to call it just "informal writing".